package.json

```diff
@@ -34,6 +34,7 @@
     "dotenv": "^16.4.7",
     "graphql": "^16.10.0",
     "i18next": "^22.5.1",
+    "is-arrayish": "0.3.2",
```

Comment:
Last version of this dependency was published 7 years ago, we can’t use such dependencies in our project
Please remove it